android/src/main/java/kr/ibigtree/rnnaverlogin/NaverLogin.kt

```diff
@@ -16,7 +16,7 @@ class NaverLoginModule(private val reactContext: ReactApplicationContext) : Reac
     }
 
     @ReactMethod
-    fun login(options: ReadableMap, promise: Promise) {
+    fun initialize(options: ReadableMap) {
```

Comment:
We need to initialize when we use login and logout. So, I made initialize method to support both.